src/System.Windows.Forms/src/System/Windows/Forms/ComboBox.cs

```diff
@@ -3903,7 +3903,7 @@ protected override void WndProc(ref Message m)
                     break;
 
                 case WM.PAINT:
-                    if (GetStyle(ControlStyles.UserPaint) == false && (FlatStyle == FlatStyle.Flat || FlatStyle == FlatStyle.Popup))
+                    if (GetStyle(ControlStyles.UserPaint) == false && (FlatStyle == FlatStyle.Flat || FlatStyle == FlatStyle.Popup) && !DisplayInformation.HighContrast)
```

Comment:
We use `SystemInformation` for it:
```suggestion
                    if (GetStyle(ControlStyles.UserPaint) == false && (FlatStyle == FlatStyle.Flat || FlatStyle == FlatStyle.Popup) && !SystemInformation.HighContrast)
```

Reply:
@vladimir-krestov - did you find any problems with `DisplayInformation`? If there are any, we should fix them or get rid of this class.

Reply:
No, I didn't. It's just for consistency. Will take a look at these classes